app/controllers/users_controller.rb

```diff
@@ -1,5 +1,28 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = [
+      { 'id' => '1', 'username' => 'peter druid', 'number_of_posts' => '3' },
+      { 'id' => '2', 'username' => 'david armstrong', 'number_of_posts' => '1' }
+    ]
+  end
 
-  def show; end
+  def show
+    @users = [
+      { 'id' => '1', 'username' => 'peter druid', 'number_of_posts' => '3' },
+      { 'id' => '2', 'username' => 'david armstrong', 'number_of_posts' => '1' }
+    ]
```

Comment:
- [ ] It would be better if we make use of the rails' active record functionality. `@user = User.find(params[:id])`. This query will help us to retrieve a specific user with a specific id